tests/detect-hostbits/detect-hostbits-01/test.yaml

```diff
@@ -0,0 +1,17 @@
+args:
+- -k none
+
+checks:
```

Comment:
we should probably also add checks for sid 1 and sid 2 not generating alerts, so `count: 0`